qiskit_experiments/library/randomized_benchmarking/rb_experiment.py

```diff
@@ -107,6 +108,13 @@ def _verify_parameters(self, lengths, num_samples):
         if num_samples <= 0:
             raise QiskitError(f"The number of samples {num_samples} should " "be positive.")
 
+        # Set configurable options
+        self.set_experiment_options(lengths=list(lengths), num_samples=num_samples, seed=seed)
```

Comment:
```suggestion
        self.set_experiment_options(lengths=sorted(lengths), num_samples=num_samples, seed=seed)
```
Likely we implicitly assume ascending order in the internal logic.